gradunwarp/core/coeffs.py

```diff
@@ -96,7 +92,15 @@ def get_siemens_coef(cfile):
 def get_ge_coef(cfile):
     ''' Parse the GE .coef file.
     '''
-    txt_var_map = create_txt_var_map(coef_array_sz)
+    # R0_m has never been defined in gradunwarp for this function
+    # This function would raise a NameError if it were ever called, so
+    # let's provide a meaningful runtime error.
+    raise RuntimeError(
+        "R0_m has never been defined in gradunwarp for this function. "
+        "If you know how to determine it, please contact the developers."
+    )
+
+    txt_var_map = create_txt_var_map(ge_cas)
```

Comment:
Two separate problems are addressed here. First, the function looks like it's never worked. Second, the `ge_cas` variable was lost in #9.

Reply:
#9 doesn't appear to have deleted a definition of `ge_cas`, and this edit uses `ge_cas` in the new code.  Clarification please?

Reply:
In 4af3c751d4f14f4e7844c2190588a7593d25979c:

```diff
 def get_ge_coef(cfile):
     ''' Parse the GE .coef file.
     '''
-    ax = np.zeros((ge_cas, ge_cas))
-    ay = np.zeros((ge_cas, ge_cas))
-    az = np.zeros((ge_cas, ge_cas))
-    bx = np.zeros((ge_cas, ge_cas))
-    by = np.zeros((ge_cas, ge_cas))
-    bz = np.zeros((ge_cas, ge_cas))
-    txt_var_map = {'Alpha_x': ax,
-                   'Alpha_y': ay,
-                   'Alpha_z': az,
-                   'Beta_x': bx,
-                   'Beta_y': by,
-                   'Beta_z': bz}
+    txt_var_map = create_txt_var_map(coef_array_sz)
 
     coef_file_parse(cfile, txt_var_map)
 
-    return Coeffs(ax, ay, az, bx, by, bz, R0_m)
+    return Coeffs(
+        txt_var_map['alpha_y'],
+        txt_var_map['alpha_z'],
+        txt_var_map['alpha_x'],
+        txt_var_map['beta_y'],
+        txt_var_map['beta_x'],
+        txt_var_map['beta_z'],
+        R0_m)
 
 def grad_file_parse(gfile, txt_var_map):
```

The arrays were defined to be `(ge_cas, ge_cas)` in size, and now they are `coef_array_sz`, an undefined variable.

Reply:
I see, you aren't saying the `ge_cas` variable stopped existing, you are saying #9 incorrectly changed it to a different symbol, which wasn't defined.

Reply:
Yes, apologies for the confusion. I discovered this because ruff complained that the `ge_cas` import from `.globals` (L12) was unused.